src/ledger-keyring.test.ts

```diff
@@ -9,6 +9,14 @@ import { LedgerBridge } from './ledger-bridge';
 import { LedgerIframeBridge } from './ledger-iframe-bridge';
 import { AccountDetails, LedgerKeyring } from './ledger-keyring';
 
+jest.mock('@metamask/eth-sig-util', () => {
+  return {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    __esModule: true,
+    ...jest.requireActual('@metamask/eth-sig-util'),
+  };
+});
+
```

Comment:
@legobeat is this necessary. Don't understand what, if anything, this is doing?

Reply:
Yes, this is a common approach when bundling all exports from an import into one variable and then spying on an individual export. Basically we're making a copy of `sigUtil` so that all properties are writable (`jest.spyOn` reassigns the property you spy on).